mantidproject/mantid
# Engg GUI::Unit Tests For Cropped Calibration

fixes #14923 

For the new type of calibration which has been just merged in #14830, few test cases have been added, which will test all the functionality of new cropped calibration.  _(A positive test case has also been included)_

**To Test**
- Run the `EnggDiffractionPresenterTest`  locally
- Check the tests covers the cropped calibration comprehensively
- Test cases also checks the inconsistency of the spectrumIDs/Banks numbers
